constantine/ethereum_evm_precompiles.nim

```diff
@@ -60,6 +60,31 @@ func eth_evm_sha256*(r: var openArray[byte], inputs: openArray[byte]): CttEVMSta
   sha256.hash(cast[ptr array[32, byte]](r[0].addr)[], inputs)
   return cttEVM_Success
 
+func modexp_result_size*(size: var uint64, inputs: openArray[byte]): CttEVMStatus {.noInline, tags:[Alloca, Vartime], libPrefix: prefix_ffi, meter.} =
```

Comment:
```suggestion
func eth_evm_modexp_result_size*(size: var uint64, inputs: openArray[byte]): CttEVMStatus {.noInline, tags:[Alloca, Vartime], libPrefix: prefix_ffi, meter.} =
```

Reply:
I wasn't sure whether to reuse the naming convention. I didn't so that there is a clearer distinction between what is a real EVM precompiles and what is a helper function. Will change it.

Reply:
It's namespacing in case several protocols have differing modexp, say a modexp for the EVM, a modexp to simulate GMP, etc ...